src/docs/santiment-queries/liquidation/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Liquidation Overview
+author: Santiment Team
+date: 2023-04-03
+description: Information about the Liquidation project's tables stored in Santiment Queries.
+---
+
+## Overview
+
+Liquidation is a project that aims to track deposits, borrows, repayments, withdrawals, and liquidation actions occurring on lending protocols.
+
+In **Santiment Queries**, we collect liquidation data from the following platforms:
+
+- AAVE-v2/v3
+- Compound
+- Liquity
+- MakerDAO
```

Comment:
In the article we need to clearly state what is the value of the `project_name` column for each project. This can be used by the AI to construct better queries